src/Umbraco.Cms.Api.Management/ViewModels/Webhook/CreateWebhookRequestModel.cs

```diff
@@ -0,0 +1,6 @@
+namespace Umbraco.Cms.Api.Management.ViewModels.Webhook;
+
+public class CreateWebhookRequestModel : WebhookModelBase
+{
+    
+}
```

Comment:
This model should have a nullable key, and we should be creating the webhook with said Id (Key in backend terms 🤪 )if its not null.

(This is for use cases like offline-mode support in the future) 👍 
```suggestion
{
    public Guid? Id { get; set; }
}
```